client/components/App.jsx

```diff
@@ -12,16 +12,16 @@ import Troubleshoot from './Troubleshoot';
 const App = () => {
 
 	const [selected, setSelected] = useState({
-		name1: true,
-		name2: true,
-		noLibrary: false,
-		react: false,
-		bootstrap: false,
+		name1: false,
+		name2: false,
+		noLibrary: true,
+		react: true,
```

Comment:
It might be giving errors due to both noLibrary and react being set to true initially. These have to be set to opposite values (true or false),